deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,93 @@
+
+
+elasticsearch:
+  host: ""
+  user: ""
+  pass: ""
+
+cloudDefend:
+  enabled: false
+
+kubernetes: 
+  version: 1.51.0
+  enabled: false
```

Comment:
Is this needed?

Reply:
hmm after running some test I have to say apparently it is not 😕